test/ConverterTest.java

```diff
@@ -0,0 +1,51 @@
+import edu.cscc.Converter;
+import edu.cscc.Employees;
+import edu.cscc.EmployeesDocument;
+import org.junit.jupiter.api.Test;
+
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBException;
+import javax.xml.bind.Unmarshaller;
+import java.io.File;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ConverterTest {
+
+    @Test
+    public void canConvertEmployees() throws JAXBException {
```

Comment:
These tests are checking for multiple things at once. Partly they're checking that the employees.xml can be marshalled correctly, and partly they're checking that the Converter produces the right output. 

The marshalling tests should be split out from the converter tests. The Converter only cares about an Employees object, not that it came from XML. To make your life easier in the Converter tests you could create an Employees object with a single Employee in it, and validate that invoking `convertEmployees()` produces an object with the correct properties.